src/pages/index.tsx

```diff
@@ -39,10 +39,10 @@ const links = [
         color: '#1099A8',
     },
     {
-        text: 'XP Table',
-        url: '/xp',
+        text: 'Epic Spells Things',
+        url: '/epic',
         badge: true,
-        description: "Here's the 5e XP table because no one ever knows these numbers",
+        description: "Knowing magic is cool This'll make you cooler.",
```

Comment:
hell yeah

Reply:
I did miss punctuation in there, so I fixed it ae4602f